task/sql-tasks.js

```diff
@@ -281,7 +439,27 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT DISTINCT
+		Customers.CompanyName as CompanyName,
+		Products.ProductName as ProductName,
+		OrderDetails.UnitPrice as PricePerItem
+	FROM Orders
+		INNER JOIN OrderDetails on Orders.OrderID = OrderDetails.OrderID
+		INNER JOIN Customers on Orders.CustomerID = Customers.CustomerID
+		INNER JOIN Products on OrderDetails.ProductID = Products.ProductID
+	WHERE OrderDetails.UnitPrice = 

```

Comment:
this is kinda slow. In subquery, try to group by customer id and join subquery on price and customer id